test/quantization/fx/test_equalize_fx.py

```diff
@@ -250,23 +253,29 @@ def test_input_weight_equalization_prepare(self):
             ns.call_module(MinMaxObserver): 6,
         }
 
-        tests = [(SingleLayerLinearModel, single_nn_layer_node_occurrence),
-                 (TwoLayerLinearModel, two_nn_layer_node_occurrence),
-                 (TwoLayerFunctionalLinearModel, two_F_layer_node_occurrence),
-                 (FunctionalLinearAddModel, fp_F_layer_node_occurrence),
-                 (LinearReluModel, single_nn_layer_node_occurrence),
-                 (LinearReluLinearModel, two_nn_layer_node_occurrence),
-                 (FunctionalLinearReluModel, single_F_layer_node_occurrence),
-                 (FunctionalLinearReluLinearModel, two_F_layer_node_occurrence),
-                 (ConvModel, single_nn_layer_node_occurrence),
-                 (TwoLayerConvModel, two_nn_layer_node_occurrence),
-                 (TwoLayerFunctionalConvModel, two_F_layer_node_occurrence),
-                 (ConvReluModel, single_nn_layer_node_occurrence),
-                 (ConvReluConvModel, two_nn_layer_node_occurrence),
-                 (FunctionalConvReluModel, single_F_layer_node_occurrence),
-                 (FunctionalConvReluConvModel, two_F_layer_node_occurrence)]
-
-        for (M, node_occurrence) in tests:
+        multiple_ops_node_occurrence = {
+            ns.call_module(_InputEqualizationObserver): 2,
+            ns.call_module(MinMaxObserver): 7,
+        }
+
+        tests = [(SingleLayerLinearModel, default_qconfig_dict, single_nn_layer_node_occurrence),
+                 (TwoLayerLinearModel, default_qconfig_dict, two_nn_layer_node_occurrence),
+                 (TwoLayerFunctionalLinearModel, default_qconfig_dict, two_F_layer_node_occurrence),
+                 (FunctionalLinearAddModel, specific_qconfig_dict, fp_F_layer_node_occurrence),
+                 (LinearReluModel, default_qconfig_dict, single_nn_layer_node_occurrence),
+                 (LinearReluLinearModel, default_qconfig_dict, two_nn_layer_node_occurrence),
+                 (FunctionalLinearReluModel, default_qconfig_dict, single_F_layer_node_occurrence),
+                 (FunctionalLinearReluLinearModel, default_qconfig_dict, two_F_layer_node_occurrence),
+                 (ConvModel, default_qconfig_dict, single_nn_layer_node_occurrence),
+                 (TwoLayerConvModel, default_qconfig_dict, two_nn_layer_node_occurrence),
+                 (TwoLayerFunctionalConvModel, default_qconfig_dict, two_F_layer_node_occurrence),
+                 (ConvReluModel, default_qconfig_dict, single_nn_layer_node_occurrence),
+                 (ConvReluConvModel, default_qconfig_dict, two_nn_layer_node_occurrence),
+                 (FunctionalConvReluModel, default_qconfig_dict, single_F_layer_node_occurrence),
+                 (FunctionalConvReluConvModel, default_qconfig_dict, two_F_layer_node_occurrence),
+                 (ModelMultipleOpsNoAvgPool, default_qconfig_dict, multiple_ops_node_occurrence)]
```

Comment:
is ModelMultipleOpsNoAvgPool testing the branch case? Maybe we can have a more directed test case for this that checks to make sure the warning was raised since `test_input_weight_equalization_prepare` seems to be getting very complex.